tests/vmi_configuration_test.go

```diff
@@ -697,7 +701,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			var vmi *v1.VirtualMachineInstance
 
 			BeforeEach(func() {
-				vmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				vmi = libvmi.NewCirros()
```

Comment:
I think you can remove the line below

Reply:
Yep, NewCirros add the same by default

Reply:
It would be good to see the `OvercommitGuestOverhead` as an option to not combine 2 different approaches to how to create VM.